### ISSUE TITLE
CJ4 0.10.1 - Throttle Keybinds

### ISSUE THREAD

**Issue by markofthemartians** [Working-Title-MSFS-Mods/fspackages#892]
I am playing with a PS4 controller and I use F1-F10 keybinds to set my throttle at 10% intervals.. (F1=10%, F2=20%...F10=100%...).

I've found that setting the throttle doesn't work as a toggle, but instead I have to hold the key down in order to achieve my desired throttle power. 

Please let me know if you need more information. Thanks.

**Comment by dga711** [comment]
I think our FADEC doesnt handle these interval keybinds because actually I never thought someone would use them ;)

Can you not bind for example L and R triggers to the Axis ?

**Comment by markofthemartians** [comment]
hey thanks for the quick response! I'm using controller bindings (X and O) for throttle up and down as a workaround, but the reason I use the F1-F10 keys is so I can be more accurate with setting the throttle power. it's easier and quicker to set 50% throttle by pressing F5 than using throttle up and down. if i had actual flight sim gear this wouldn't even be an issue, but until I get one, this is the best way I've found when playing with controller.

**Comment by dga711** [comment]
Yeah, i see. We can look into also making these events available.
